doc-content/enterprise-only/cases/case-management-clean-case-log-proc.adoc

```diff
@@ -13,49 +13,53 @@ There are several important configuration options that can be used with the `Cas
 | Is Exclusive
 
 | SkipProcessLog
-| Indicates if the clean-up of process instances, node instances and variables log cleanup should be omitted (default: false)
+| Indicates if the of process and node instances, and variable logs clean-up should be omitted. (default: false)
```

Comment:
Probably typo in `Indicates if the of process and node instances`.

Reply:
Fixed. Thank you!